package.json

```diff
@@ -24,7 +24,7 @@
   "dependencies": {
     "@actions/core": "^1.10.1",
     "@actions/github": "^6.0.0",
-    "@octokit/core": "^5.1.0",
+    "@octokit/core": "^6.0.0",
     "@octokit/graphql-schema": "^14.57.0",
     "@octokit/plugin-paginate-graphql": "^4.0.0",
```

Comment:
https://github.com/octokit/plugin-paginate-graphql.js/pull/166#pullrequestreview-1887037767

need to wait the updates for plugin-paginate-graphql? 👀